Dockerfile

```diff
@@ -1,11 +1,16 @@
 FROM docs/docker.github.io:docs-base
 
+# IF THIS IS NOW AN ARCHIVE: remove everything in this file (including the line above) and replace it with the below (and uncomment):
+# (see https://docker.atlassian.net/wiki/spaces/DOCSTEAM/pages/133081331/Docs+release+and+archiving+process):
+#
+# FROM starefossen/github-pages:137
```

Comment:
Soon to be 147 just FYI; https://github.com/docker/docker.github.io/pull/5241

There's not a hard dependency between that PR and this one but it might be prudent to just change it to 147